econml/cate_estimator.py

```diff
@@ -37,7 +85,9 @@ def fit(self, Y, T, X=None, W=None, Z=None):
         self
 
         """
-        pass
+        if hasattr(self, '_bootstrap'):
```

Comment:
Does this mean that it automatically fits the bootstrap CI at fit time? That would be terrible for some of the slow algos, such as the ORF.